host_core/lib/application.ex

```diff
@@ -45,6 +45,11 @@ defmodule HostCore.Application do
     started = Supervisor.start_link(children, opts)
 
     if config.enable_structured_logging do
+      :logger.set_primary_config(
+        :logger.get_primary_config()
+        |> Map.put(:level, config.structured_log_level)
+      )
```

Comment:
A little annoying that this is required but I accept it, looks like https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/config/config.exs#L20 is something set at _release time_ not at _run time_, right?

Reply:
I think that's correct, based off this: https://elixir-lang.org/getting-started/mix-otp/config-and-releases.html#configuration